tests/Makefile

```diff
@@ -57,7 +58,8 @@ e2e-test:
 	CLUSTER_DEFINITION_URL=$(CLUSTER_DEFINITION_URL) scripts/docker-test.sh
 
 clean-after-tests:
-	rm -rf test_log/ tmp/ allure/ .dp test-result.html test-result.xml
+	sudo chmod -Rf 777 tmp/
+	rm -rf test_log/ tmp/ allure/ .dp test-result.html test-result.xml swagger-*.json
```

Comment:
I've just realized the [CircleCI Artifacts](https://github.com/hortonworks/cb-cli/blob/master/.circleci/config.yml#L15-L20) are not generated because of this. Could you change to `rm -rf test_log/ tmp/ .dp test-result.xml swagger-*.json`?